archey

```diff
@@ -479,6 +500,11 @@ class Output:
             exit()
 
         for distribution in Distributions:
+            if re.fullmatch(lsbOutput, "openSUSE") or re.fullmatch(lsbOutput, "Ubuntu"):
```

Comment:
Hey Roy, I'm ready ! @Si13n7

Could you paste here the different outputs of `lsb_release -i -s` and `uname -r` ?
I'd like to optimize this very loop without this test case before merging this PR !

Thx again,
👋

Reply:
Hey Samuel,

here:

| System   | lsb_release -i -s | uname -r           |
| --------- | ----------------- | ----------------- |
| Ubuntu 16.04.3 LTS | Ubuntu | 4.4.0-109-generic  |
| Windows 10 + Ubuntu Subsystem | Ubuntu | 4.4.0-43-Microsoft |

Reply:
Thanks ! I guess for `openSUSE` case it'd be the same as the second row of your table by replacing `Ubuntu` ?

Reply:
I don't have an openSUSE installation right now, but I still remember that when I tested it the outputs were similar 😄

Reply:
Well okay 😂
I'll start working on this this evening. 'keep you posted 😉

Reply:
I added the openSUSE subsystem to the list 😉

Reply:
Ahhh thanks, so fast 😜

Reply:
Well... I can't push to your fork this time... Do you have a checkbox somewhere around here to allow me to contribute to the fork ? 😕

Reply:
I can't see anything like that...

Reply:
Wtf 😂 Could you send me a new invitation for this time ? 🙂

Reply:
EDIT: [link removed] (check your emails 😜)

Reply:
Thanks it's pushed 😜
Tell me what you think about this, I think it's better to check for `Microsoft` anyway...

Reply:
lets measure and compare...

| old | new |
| --- | --- |
| 0.12305116653442383s | 0.11970376968383789s |

Yeah, your way is better! 😜